.github/workflows/UpdateContentOverview.yml

```diff
@@ -52,3 +58,13 @@ jobs:
           token: ${{ secrets.GH_PAT }}
           pull-request-number: ${{ steps.cpr.outputs.pull-request-number }}
           merge-method: merge
+      - name: "Setting AWS credentials"
```

Comment:
This action shouldn't be allowed to execute for forked PRs (If someone fork and then change files in scripts/data/**/*). Please take a look in e2e test to prevent it.

Reply:
Do you mean this check?
              `if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository`

I was under the impression that GH actions for outside contributors require explicit approval, always, and that that would be enough. But you're probably right, best to add another layer.

Reply:
Yeap, it will only execute when approving. But forked PRs doesnt have access to secrets so those steps will fail.

Reply:
Done